### DISCUSSION TITLE
Why doesn't Noticed use the official Twilio gem?

### DISCUSSION THREAD

**Discussion by smarquez1** [excid3/noticed#511] [Q&A] [answered]
It caught my attention that [Noticed::DeliveryMethods::TwilioMessaging](https://github.com/excid3/noticed/blob/main/lib/noticed/delivery_methods/twilio_messaging.rb) calls the Twilio API directly instead of using the official [Twilio gem](https://github.com/twilio/twilio-ruby). I'm curious, what's the reasoning behind this?

**Comment by excid3** [comment, marked as answer]
Less dependencies.
